src/lld/dpdk/Makefile.am

```diff
@@ -2,7 +2,7 @@ all:
 	$(shell ./apply_patch.sh > /dev/null 2>&1)
 	stat dpdk_src/build > /dev/null 2>&1 || \
 	(cd dpdk_src && meson -Dprefix=$(prefix) build && \
-	cd build && ninja -j48 && ninja install)
+	cd build && ninja && ninja install)
```

Comment:
just say ninja -j ; without number else the builds are very slow

Reply:
it looks like only ninja -j is not working, it expects an argument.
  "ninja: option requires an argument -- 'j'"